src/components/recruting/_02_prepare/_01/NumberSpinner.tsx

```
@@ -0,0 +1,79 @@
+            const currentValue = value || 0;
+            if (currentValue <= 0) return;
+            onChange(currentValue - 1);
+            onBlur();
+          };
+
+          const handleInputChange = (
+            e: React.ChangeEvent<HTMLInputElement>
+          ) => {
+            const newValue =
+              e.target.value === "" ? undefined : parseInt(e.target.value);
+            if (isNaN(newValue as number) || (newValue as number) < 0) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)